test/button.html

```diff
@@ -1,108 +1,82 @@
-					<td>Normal</td>
-					<td>Input</td>
-					<td>Mixin</td>
-					<td><input type="button" value="Button" /></td>
-					<td><input type="button" value="Button" disabled /></td>
-					<td><input type="button" value="Button" class="vui-button-focus" /></td>
-				</tr>
-				<tr>
-					<td>Normal</td>
-					<td>Input</td>
-					<td>Class</td>
-					<td><input type="button" value="Button" class="vui-button" /></td>
-					<td><input type="button" value="Button" class="vui-button" disabled /></td>
-					<td><input type="button" value="Button" class="vui-button vui-button-focus" /></td>
-				</tr>
-				<tr>
-					<td>Normal</td>
-					<td>Anchor</td>
-					<td>Class</td>
-					<td><a class="vui-button">Button</a></td>
-					<td><a class="vui-button" disabled>Button</a></td>
-					<td><a class="vui-button vui-button-focus">Button</a></td>
-				</tr>
-				<tr class="on">
-					<td>Primary</td>
-					<td>Button</td>
-					<td>Mixin</td>
-					<td><button class="primary">Button</button></td>
-					<td><button class="primary" disabled>Button</button></td>
-					<td><button class="primary vui-button-focus">Button</button></td>
-				</tr>
-				<tr>
-					<td>Primary</td>
-					<td>Button</td>
-					<td>Class</td>
-					<td><button class="vui-button-primary">Button</button></td>
-					<td><button class="vui-button-primary" disabled>Button</button></td>
-					<td><button class="vui-button-primary vui-button-focus">Button</button></td>
-				</tr>
-				<tr class="on">
-					<td>Primary</td>
-					<td>Input</td>
-					<td>Mixin</td>
-					<td><input type="button" value="Button" class="primary" /></td>
-					<td><input type="button" value="Button" class="primary" disabled /></td>
-					<td><input type="button" value="Button" class="primary vui-button-focus" /></td>
-				</tr>
-				<tr>
-					<td>Primary</td>
-					<td>Input</td>
-					<td>Class</td>
-					<td><input type="button" value="Button" class="vui-button-primary" /></td>
-					<td><input type="button" value="Button" class="vui-button-primary" disabled /></td>
-					<td><input type="button" value="Button" class="vui-button-primary vui-button-focus" /></td>
-				</tr>
-				<tr>
-					<td>Primary</td>
-					<td>Anchor</td>
-					<td>Class</td>
-					<td><a class="vui-button-primary">Button</a></td>
-					<td><a class="vui-button-primary" disabled>Button</a></td>
-					<td><a class="vui-button-primary vui-button-focus">Button</a></td>
-				</tr>
-			</tbody>
-		</table>
-		<p class="screenshot on" style="display:inline-block;">
-			<button class="primary">Primary Button</button>&nbsp;
-			<button>Normal Button</button>&nbsp;
-			<button disabled>Disabled Button</button>
-		</p>
+		<test-fixture id="NormalButton">
+			<template>
+				<button is="vui-button">Button</button>
+			</template>
+		</test-fixture>
+		<test-fixture id="PrimaryButton">
+			<template>
+				<button is="vui-button" primary>Button</button>
+			</template>
+		</test-fixture>
+		<script>
+		suite('<vui-button>', function() {
```

Comment:
I wasn't sure if we should use BDD or TDD style tests - I think in my experimenting I used the BDD stlye.  I don't feel strongly about either one, accept that we should try to be consistent.